backend/routes/appointment.js

```diff
@@ -179,6 +219,19 @@ router.put("/change-appointment-status", async (req, res) => {
       },
     });
 
+    if (
+      status === "CANCELLED" ||
+      status === "RESCHEDULED" ||
+      status === "COMPLETED"
```

Comment:
nit: Try to avoid hard coded string in the code, maybe put them into const strings.